### PR TITLE
#3188 : Group > Private message > All the users are not displayed

### LINKED ISSUE #3188: Group > Private message > All the users are not displayed
**Describe the bug**
Total users does not appear in the private message section list.

**To Reproduce**
1. Go to 'Group private message'
2. Scroll down 'user list'

**Expected behavior**
All the users should appear in the message list.

**Screenshots**
https://www.loom.com/share/4e562eb0b5294614a3dbf01bfadd833a

**Environment**
Local / Demo

### PR DESCRIPTION
**Pull request by milanhirapra**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #3188 .

### How to test the changes in this Pull Request:

1. Go to 'Group private message'
2. Scroll down 'user list'
3. See error

**NOTE:** If users blocked then it does affect on total user count. 
- Sumit mentioned in this issue, The group has 1251 users and Send message screen appears 1221 count. 
=> The reason is bp_group_members table has duplicate user entry so, the total count appears 1251 instead of actual user count.

### Proof Screenshots or Video
https://vimeo.com/649891810/8586f36d9c

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-templates/bp-nouveau/js/buddypress-group-messages.js [hunk 1]
```diff
@@ -854,6 +854,10 @@ window.bp = window.bp || {};
 			);
 		},
 
+		getMemberOnScroll: function() {
+
+		},
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/js/buddypress-group-messages.js:859]: @milanhirapra Not needed

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Remove `getMemberOnScroll` blank function as no needed.

File: src/bp-templates/bp-nouveau/js/buddypress-group-messages.js [hunk 2]
```diff
@@ -302,7 +302,7 @@ window.bp = window.bp || {};
 				var membersDiv = document.getElementById( 'members-list' );
 				$( '.bb-icon-loader' ).hide();
 				var scroll_xhr;
-				if ( membersDiv.length ) {
+				if ( $(membersDiv).length ) {
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/js/buddypress-group-messages.js:305]:
```suggestion
				if ( $( membersDiv ).length ) {
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need changes

**Review by chetansatasiya — APPROVED** [review]

**Comment by sumitbuddyboss** [comment]
Hello @milanhirapra 

Fixed >
Now all the members are displayed for the proof please check the below video.

Video:
https://www.loom.com/share/0ee419056b3a473fa9aed1879e89c7fe

Test Result:  Pass

Ready for release 🚀 

Thank you

**Comment by basil-buddyboss** [comment]
Hi @milanhirapra 

This has been tested in the QA-approved branch without any issues encountered.

Result: Pass

**Pull request merged**